examples/javascript/gpio-tool.js

```diff
@@ -61,47 +61,48 @@ function onPinLevelChange() {
 }
 
 function monitorPin() {
-    var pinNumber = arguments[0];
+    let pinNumber = arguments[0];
     try {
-        var pin = new mraa.Gpio(pinNumber);
+        let pin = new mraa.Gpio(pinNumber);
         pin.dir(mraa.DIR_IN);
         pin.isr(mraa.EDGE_BOTH, onPinLevelChange);
         rl.question('Press ENTER to stop', function(answer) {
             rl.close();
             pin.isrExit();
         });
     } catch (err) {
-     console.log(err.message);
+        console.log(err.message);
     }
 }
 
 const args = process.argv;
 const argc = args.length;
+
 if (argc >= 3) {
     const cmd = args[2];
+    if (argc >= 3) {
```

Comment:
Ok, this one should simply be converted to `if (argc > 3)`, then it starts making sense, because pin number only appears as a fourth member in `args`.